Sources/Contentful/Client.swift

```diff
@@ -458,13 +452,10 @@ open class Client {
         """
         ContentfulLogger.log(.error, message: errorMessage)
         // In this case, .success means that a RateLimitError was successfully initialized.
-        completion(Result.success(rateLimitError))
+        return .success(rateLimitError)
```

Comment:
This is an example where the `return` was missing, so the execution would continue after calling `completion` and would result in `completion` being called multiple times.